massgov/mayflower
# DP-19354 Search button behavior

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-19354)
- [Github issue]()

## Steps to Test

Set the behavior to put focus on the search input in the hamburger menu by clicking the search button on the blue bar when the menu is open.

1. Go to http://localhost:3000/patterns/05-pages-Homepage-hamburger/05-pages-Homepage-hamburger.html.
2. In the window width of 620px and smaller, click the search button on the blue bar and check its behavior:
- When the search button is clicked while the menu is CLOSE, the menu opens and focus is set to the search input box.
- When the search button is clicked while the menu is OPEN, focus is set to the search input box. Anything else doesn't change.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
